src/pages/index.module.scss

```diff
@@ -58,6 +58,10 @@
   justify-content: center;
   margin: 0 auto 5em auto;
 
+  @media (max-width: 870px) {
```

Comment:
Why 870px? We do have a variable that we use across the website. I think it is either 600px or 900px 🤔 Could you double-check?

Reply:
Oh, I saw `870px`  being used for hiding the demo container [here](https://github.com/nodejs/nodejs.dev/blob/main/src/pages/index.module.scss#L4) and thought of just using it again

Reply:
I also saw that the navbar isn't fixed so that explains why it keeps moving towards the left side on smaller viewports

Reply:
> Oh, I saw 870px being used for hiding the demo container [here](https://github.com/nodejs/nodejs.dev/blob/main/src/pages/index.module.scss#L4) and thought of just using it again

Probably we should update that one too.

Reply:
900px is being used across the website but I don't think there's a variable for it

Reply:
Yup, just use 900px

Reply:
Updated it to be 900px